README.md

```diff
@@ -1,13 +1,131 @@
+Go to the developer console of your Salesforce account and click on **Debug->Open Execute Anonymous Window.** Add the following entry in the Enter Apex Code window. 
+
+```
+PushTopic pushTopic = new PushTopic();
+pushTopic.Name = 'InvoiceStatementUpdates';
+pushTopic.Query = 'SELECT Id, Name, wso2__Status__c, wso2__Description__c FROM InvoiceStatement__c';
+pushTopic.ApiVersion = 37.0;
+pushTopic.NotifyForOperationCreate = true;
+pushTopic.NotifyForOperationUpdate = true;
+pushTopic.NotifyForOperationUndelete = true;
+pushTopic.NotifyForOperationDelete = true;
+pushTopic.NotifyForFields = 'Referenced';
+insert pushTopic;
+```
+Click **Execute**.
+
+
+## Retrieving the Account Information 
+WSO2 ESB Salesforce inbound endpoint acts as a message consumer. It creates a connection to the Salesforce account, consumes the Salesforce data and injects the data to the ESB sequence.
+Now that you have configured the Salesforce streaming inbound endpoint, use the following ESB inbound endpoint configuration to retrieve account details from your Salesforce account.
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<inboundEndpoint xmlns="http://ws.apache.org/ns/synapse"
+                 name="SaleforceInboundEP"
+                 sequence="test"
+                 onError="fault"
+                 class="org.wso2.carbon.inbound.salesforce.poll.SalesforceStreamData"
+                 suspend="false">
+   <parameters>
+      <parameter name="inbound.behavior">polling</parameter>
+      <parameter name="interval">100</parameter>
+      <parameter name="sequential">true</parameter>
+      <parameter name="coordination">true</parameter>
+      <parameter name="connection.salesforce.replay">false</parameter>
+      <parameter name="connection.salesforce.EventIDStoredFilePath">/Users/nalaka/Desktop/a.txt</parameter>
+      <parameter name="connection.salesforce.packageVersion">37.0</parameter>
+      <parameter name="connection.salesforce.salesforceObject">/topic/InvoiceStatementUpdates</parameter>
+      <parameter name="connection.salesforce.loginEndpoint">https://login.salesforce.com</parameter>
+      <parameter name="connection.salesforce.userName">MyUsername</parameter>
+      <parameter name="connection.salesforce.password">MyPassword</parameter>
+      <parameter name="connection.salesforce.waitTime">5000</parameter>
+      <parameter name="connection.salesforce.connectionTimeout">20000</parameter>
+      <parameter name="connection.salesforce.soapApiVersion">22.0</parameter>
+   </parameters>
+</inboundEndpoint>
+```
+* connection.salesforce.replay: replay enable or disable. If this enabled read from event id stored in registry DB or from the text file stored in the local machine.
```

Comment:
If you can highlight (BOLD) the attributes, It will be more clear.

Reply:
fixed